src/views/VenueDetails/VenueDescription/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import Rating from '@mui/material/Rating'
+import Carousel from 'react-material-ui-carousel'
+import styles from './styles.module.css'
+import { VenueImgContainer } from './VenueImgContainer'
+import { Description } from './Description'
+import { Gallery } from './Gallery'
+import { LocationMap } from './LocationMap'
+
+export function VenueDescription ({ venueDetails }) {
+  const [displayDescription, setDisplayDescription] = useState(true)
+  const [displayGallery, setDisplayGallery] = useState(false)
+  const [displayMap, setDisplayMap] = useState(false)
+
+  let imgsToDisplay = venueDetails.venueDescriptionImgUrls
+  imgsToDisplay = { 0: venueDetails.landingImgUrl, ...imgsToDisplay }
+  imgsToDisplay = Object.values(imgsToDisplay)
```

Comment:
If `venueDescriptionImgUrls` is an array, then we can do this…
```suggestion
  let imgsToDisplay = venueDetails.venueDescriptionImgUrls;
  imgsToDisplay = [ venueDetails.landingImgUrl, ...imgsToDisplay ];
```

Reply:
It's an object tho. Should I change the data?